toolbox/incari/object/get-alpha.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+![The Get Alpha Node.](../../../.gitbook/assets/toolbox/incari/object/get-alpha.PNG)
+
+| Input | Output |
+| :--- | :--- |
+| Object ID | Alpha |
+
+**Get Alpha** returns the *alpha* value of the color space of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**. 
```

Comment:
I would maybe mention that this node only works with objects that have the "Sprite" category of attributes (List, Image Sequence, Web Sprite, Sprite, Text)